sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -178,12 +205,18 @@ export class KeyClient {
     listKeyVersions(name: string, options?: GetKeysOptions): PagedAsyncIterableIterator<KeyProperties, KeyProperties[]>;
     readonly pipeline: ServiceClientOptions;
     purgeDeletedKey(name: string, options?: RequestOptions): Promise<void>;
-    recoverDeletedKey(name: string, options?: RequestOptions): Promise<Key>;
     restoreKeyBackup(backup: Uint8Array, options?: RequestOptions): Promise<Key>;
     updateKey(name: string, keyVersion: string, options?: UpdateKeyOptions): Promise<Key>;
     readonly vaultEndpoint: string;
 }
 
+// @public
+export interface KeyPollerOptions {
+    intervalInMs?: number;
+    requestOptions?: coreHttp.RequestOptionsBase;
+    resumeFrom?: string;
+}
+
 // @public
 export interface KeyProperties extends ParsedKeyVaultEntityIdentifier {
     readonly created?: Date;
```

Comment:
`createdOn`

Reply:
This won't be done as part of this PR. The issue tracking this change is #5577